components/lists/FeedList.tsx

```diff
@@ -8,8 +16,90 @@ import { View } from 'react-native';
  * - add tests for the feed list component
  */
 
+interface FeedListItemProps {
+  item: Feed;
+}
+
+const { height } = Dimensions.get('screen');
+
+const FeedListItem = ({ item }: FeedListItemProps) => {
+  const { colors } = useTheme();
+  return (
+    <View style={styles.feed} testID="feedItem">
+      <StyledText size="md">{item.body}</StyledText>
```

Comment:
Consider truncating this text and probably adding a `read more` or `show less` touchable. This would be good in the instance that we have long body text.